### DISCUSSION TITLE
Newbie question: how to recover from an undesirable beets import?

### DISCUSSION THREAD

**Discussion by kiat-huang** [beetbox/beets#4664] [General]
Hi,

I imported a bunch of mp3s and beets suggested an album but I misunderstood and thought it was an album for one of the tracks and not all that I had selected.  Unfortunately I had chosen "move" instead of copy in the config - and so am not able to repeat the operation on a copy of the original files.

Result: files that have been renamed with Artists and Titles which are utterly different with the matching incorrect metadata.  Is there anyway to correct this so I can get beets to suggest and write the correct metadata?

**Comment by sampsyo** [comment]
You could consider doing `beet import -tL <name of album>`. This will re-import the relevant album, and do it in *timid* mode (`-t`), which means it will always ask you for confirmation instead of doing anything it thinks is "obvious." Then you can interactively choose to search for the right album/artist name at the relevant prompts.
